src/display.rs

```diff
@@ -488,15 +488,17 @@ impl Display {
 
     /// Adjust the IME editor position according to the new location of the cursor
     pub fn update_ime_position(&mut self, terminal: &Term) {
-        use crate::index::{Column, Line, Point};
-        use crate::term::SizeInfo;
-        let Point{line: Line(row), col: Column(col)} = terminal.cursor().point;
-        let SizeInfo{cell_width: cw,
-                    cell_height: ch,
-                    padding_x: px,
-                    padding_y: py, ..} = *terminal.size_info();
-        let nspot_y = (py + (row + 1) as f32 * ch) as i32;
-        let nspot_x = (px + col as f32 * cw) as i32;
+        let point = terminal.cursor().point;
+        let SizeInfo{
+            cell_width: cw,
+            cell_height: ch,
+            padding_x: px,
+            padding_y: py, ..
```

Comment:
The `..` should be on the next line.

Reply:
Is there a style guide? You don't seem to run rustfmt, and I'm not familiar with rust conventions.

Reply:
I haven't seen a style guide for this repo, but generally all Rust examples I've seen, and more generally across languages it's rare to put all items separated by a comma on separate lines except the last one, even if it's an `..` like thing.

The way it is now, it's too easy to miss.

Reply:
FWIW, there are many Rust style guide in the community, the most official (and incomplete) probably being https://doc.rust-lang.org/1.0.0/style/, however it doesn't mention this case.

Reply:
Generally I try to follow `rustfmt` and I plan to run it on the complete repo some time in the future. However since there are constantly new PRs coming in, this is extremely difficult to do.

Reply:
As long as you're asking people to do things manually there are going to be differences and omissions. In this case I would normally have put .. on its own line but since I had to format it manually I missed it, which is why I asked if there were a style guide so I could at least have a set of rules to keep in mind. 

@nixpulvis Do you mean now as in how the code is now, or now as in how it was when this comment thread started?

Reply:
> As long as you're asking people to do things manually there are going to be differences and omissions

Which is why I've pointed it out in the review. It does put an additional strain on the review process and it's easier to miss things, however just running `rustfmt` on everything isn't as simple as it might sound at first.

Though the plan to run `rustfmt` came up originall when there were like 40/50+ open PRs. So we're getting closer and closer to an acceptable number of pending changes to put through something like this.

Reply:
I only skimmed the documentation since you asked me to make this three line change myself, so I did miss that. But since you're planning to run rustfmt anyway I'm not sure it was worth the time to restyle existing code in such a small CL.